index.js

```diff
@@ -125,6 +125,9 @@ sassLint.lintText = function (file, options, configPath) {
   }
 
   if (ast.content && ast.content.length > 0) {
+
+    ruleToggles = getToggledRules(ast);
```

Comment:
There's still an issue in this file as far as I can see though

https://github.com/saptar/sass-lint/blob/5c13bf5300d58628decaa90283927221c0dbef06/index.js#L108

ruleToggles etc are passing a blank object as `ast` here, these declarations should be empty as part of this PR.